util/pace/util/grid/eta.py

```diff
@@ -35,820 +38,33 @@ def set_hybrid_pressure_coefficients(km: int) -> HybridPressureCoefficients:
-                0.00000000,
-            ]
-        )
 
-        bk = np.array(
-            [
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                0.00000000,
-                3.50123992e-06,
-                2.81484008e-05,
-                9.38666999e-05,
-                2.28561999e-04,
-                5.12343016e-04,
-                1.04712998e-03,
-                1.95625005e-03,
-                3.42317997e-03,
-                5.58632007e-03,
-                8.65428988e-03,
-                1.27844000e-02,
-                1.81719996e-02,
-                2.49934997e-02,
-                3.34198996e-02,
-                4.36249003e-02,
-                5.57769015e-02,
-                7.00351968e-02,
-                8.65636021e-02,
-                0.105520003,
-                0.127051994,
-                0.151319996,
-                0.178477004,
-                0.208675995,
-                0.242069006,
-                0.278813988,
-                0.319043010,
-                0.362558991,
-                0.408596009,
-                0.456384987,
-                0.505111992,
-                0.553902984,
-                0.601903021,
-                0.648333013,
-                0.692534983,
-                0.733981013,
-                0.772292018,
-                0.807236016,
-                0.838724971,
-                0.866774976,
-                0.891497016,
-                0.913065016,
-                0.931702971,
-                0.947658002,
-                0.961175978,
-                0.972495019,
-                0.981844008,
-                0.989410996,
-                0.995342016,
-                1.00000000,
-            ]
-        )
+    # set path where the eta file lives
+    GRID_DIR = os.path.join(os.path.abspath("./"), "input/")
```

Comment:
Instead of hardcoding the ak and bk files to `path/input/etaX.txt` we should pass the location of the file through the yaml config, similar to how Frank is setting up the model to read grid files. I'm happy to chat with you about how to do this

Reply:
Mi Kyung and I have been discussing this and plan to come up with a good location to place all necessary testing/example files.